app/Resources/translations/messages.en.xliff

```diff
@@ -246,7 +246,9 @@ This profile page gives you insight in which personal data, provided by your ins
       </trans-unit>
       <trans-unit id="c4b69512880a8532d9740e870ed8a60b8791b3bc" resname="profile.my_connections.orcid.description">
         <source>profile.my_connections.orcid.description</source>
-        <target>ORCID is a nonproprietary alphanumeric code to uniquely identify scientific and other academic authors. This addresses the problem that a particular author's contributions to the scientific literature or publications in the humanities can be hard to recognize as most personal names are not unique, they can change (such as with marriage), have cultural differences in name order, contain inconsistent use of first-name abbreviations and employ different writing systems.</target>
+        <target>Connect your existing or newly created ORCID iD to SURFconext once. The ORCID iD is a code that is used to uniquely identify scientific and other academic authors.
```

Comment:
@domgon we updated the ORCID iD pre-connection text. We managed to update our Dutch and English translations but did not even attempt a Portuguese update. Would you be so kind as to submit an improved version of this text? If you do not see any improvements whatsoever, we are fine with that too of course!

Reply:
Thanks @domgon Yes that is the first part of the sentence. The entire translation was changed. The 'target' translation of 'source' with identifier: `profile.my_connections.orcid.description`. 

Sorry for not being more clear in the first place :)